tensorflow_addons/losses/giou_loss.py

```diff
@@ -58,18 +58,9 @@ def __init__(
         reduction: str = tf.keras.losses.Reduction.AUTO,
         name: Optional[str] = "giou_loss",
     ):
-        if mode not in ["giou", "iou"]:
-            raise ValueError("Value of mode should be 'iou' or 'giou'")
-        super().__init__(name=name, reduction=reduction)
-        self.mode = mode
-
-    def get_config(self):
-        base_config = super().get_config()
-        base_config["mode"] = self.mode
-        return base_config
-
-    def call(self, y_true, y_pred):
-        return giou_loss(y_true, y_pred, mode=self.mode)
+        super(GIoULoss, self).__init__(
```

Comment:
```suggestion
        super().__init__(
```

Adding arguments to the `super` function is only needed in python 2.

Reply:
I remove them later.